pkg/executor/point_get_test.go

```diff
@@ -429,3 +429,39 @@ func TestWithTiDBSnapshot(t *testing.T) {
 
 	tk.MustQuery("select * from xx").Check(testkit.Rows("1", "7"))
 }
+
+func TestGlobalIndexPointGet(t *testing.T) {
+	defer config.RestoreFunc()()
+	config.UpdateGlobal(func(conf *config.Config) {
```

Comment:
The config is deleted, you could use session variable `tidb_enable_global_index` instead.